src/state/GlobalState.tsx

```diff
@@ -79,7 +81,7 @@ export class GlobalState extends PureComponent<Props> {
     return games
   }
   state: StateProps = {
-    category: 'epic',
+    category: storage.getItem('category') || 'epic',
```

Comment:
we can load all these local storage data here instead of in the componentDidMount callback

with the previous code, if you close heroic when in the gog library, when you open heroic it will open it in the epic library for a moment and then change to gog, which looked glitchy